README.md

```diff
@@ -1,87 +1,175 @@
+
+To pass props to a component, you add them as attributes when you render them:
 
-module.exports = MyComponent
+```js
+<MyComponent propName={propValue} />
 ```
 
-and then we can cover `React.createClass()`:
+The value of a prop is passed in through curly braces, like above. As we saw before, this value can be anything: a
+variable, inline values, functions, ... If your value is a hardcoded string, you can pass it in through double quotes
+instead:
 
-```javascript
-const React = require('react')
+```js
+<MyComponent propName="propValue" />
+```
 
-module.exports = React.createClass({
-  getDefaultProps() {
-    return {
-      greeting: 'Hello, world!'
-    }
-  },
+Armed with that knowledge, let's update our `ReactDOM.render()` call to include the data for the Mad Max movie in our
+props:
+
+```js
+ReactDOM.render(
+  <MovieCard title="Mad Max: Fury Road" poster="http://image.tmdb.org/t/p/w342/kqjL17yufvn9OVLyXYpvtyrFfak.jpg" genres={['Action', 'Adventure', 'Science Fiction', 'Thriller']} />,
+  document.getElementById('main')
+);
+```
+
+There's a small code style issue at play here: our line with the `MovieCard` component is super long, resulting in code
+that is pretty hard to read. Thankfully, we can add line breaks to make things more readable again:
+
+```js
+ReactDOM.render(
+  <MovieCard
+    title="Mad Max: Fury Road"
+    poster="http://image.tmdb.org/t/p/w342/kqjL17yufvn9OVLyXYpvtyrFfak.jpg"
+    genres={['Action', 'Adventure', 'Science Fiction', 'Thriller']} 
+  />,
+  document.getElementById('main')
+);
+```
+
+Notice how we passed in the genres as an inline array? We could also pass in variables instead, like this:
+
+```js
+const madMaxGenres = ['Action', 'Adventure', 'Science Fiction', 'Thriller'];
+
+ReactDOM.render(
+  <MovieCard
+    title="Mad Max: Fury Road"
+    poster="http://image.tmdb.org/t/p/w342/kqjL17yufvn9OVLyXYpvtyrFfak.jpg"
+    genres={madMaxGenres} 
+  />,
+  document.getElementById('main')
+);
+```
+
+This would also work for the `title` and `poster` props, but you get the idea.
+
+## Accessing props
+Now that we've passed in our props, let's change our hardcoded data in the `render()` method to make use of the props we
+pass in instead:
 
+```js
+class MovieCard extends React.Component {
   render() {
     return (
-      <div>
-        <h1>{this.props.greeting}</h1>
+      <div className="movie-card">
+        <img src={this.props.poster} alt={this.props.title} />
+        <h2>{this.props.title}</h2>
+        <small>Genres: {this.props.genres.join(', ')}</small>
       </div>
-    )
+    );
   }
-})
+}
 ```
 
-As a bonus, and since this lesson will probably be short, this might be a good
-time to introduce [ES class properties](http://babeljs.io/docs/plugins/transform-class-properties/)
+And that's all there is to it! For demonstrative purposes, let's render a movie card for Jurassic World:
```

Comment:
"For demonstrative purposes" -> "To illustrate" or something similar. The word demonstrative interestingly actually means something else...